tests/is/L42/tests/L42Source/TestGuiBuilder/This.L42

```diff
@@ -13,7 +13,7 @@ DeployGit = LoadDeploy.with(writer=GW)
 
 TaskDeployGuiBuilder=DeployGit.module(Module()
   name='Top
-  on=Url"github.com/Language42/is/GuiBuilder.L42"
+  on=Url"github.com/%(Secret.repo())/GuiBuilder.L42",
   writer=GW.#$of(
```

Comment:
You do not need the (), you can just write %Secret.repo()
42 allows for any sequence of "." stuff after the %